locale/en_US/manager.po

```diff
@@ -1104,3 +1104,15 @@ msgstr "Invalid XML:"
 
 msgid "plugins.importexport.native.exportSubmissions"
 msgstr "Export Submissions"
+
+msgid "plugins.importexport.common.cliError"
```

Comment:
This is used in the `pkp-lib` PR, but only defined in the OMP one. This (and other strings here) should probably be moved to pkp-lib.

Reply:
Removed from OMP along with `plugins.importexport.common.error.unknownContext`